eslint-plugin-bbva/.eslintrc.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
+	root: true,
+	env: {
+		node: true
+	},
+	parserOptions: {
+		ecmaVersion: 2017,
+		sourceType: 'module'
+	},
+	globals: {
+		Set: true
+	},
+	rules: [
+		'../eslint-config-bbva/rules/basic.json',
```

Comment:
Why not including all json files in rules folder? Because maybe you could forgot to add here.
Something like this:

```
const glob = require( 'glob' )
 const path = require( 'path' );

const  rules= {};
glob.sync( '../eslint-config-bbva/rules/*.json' ).forEach( ( file ) => {
   Object.assign(rules, require(path.resolve(file)).rules))
});
```